src/python/WMCore/Services/ReqMgr/ReqMgr.py

```diff
@@ -118,7 +118,7 @@ def getRequestByStatus(self, statusList):
         TODO: need proper error handling if status is not 200 from orignal reporting.
         """
         
-        query = self._createQuery({'status': statusList})
+        query = self._createQuery({'status': statusList, 'detail': detail})
```

Comment:
Doesn't it return details by default? Maybe it would be better to accept a mask, such that it returns only the RequestName and RequestStatus to reduce the memory footprint.